packages/cmrjs/tests/test-index.js

```diff
@@ -115,3 +115,30 @@ test('get CMR metadata, fail', async (t) => {
 
   stub.restore();
 });
+
+test('searchConcept handles paging correctly.', async (t) => {
```

Comment:
When stubbing http responses, I think that using [nock](https://github.com/nock/nock#readme) is a better option than sinon.  The sinon solution is more tightly coupled to the implementation.  If you decide to refactor `searchConcept` to use the `request` library instead of the `got` library, or if you use some call other than `got.get()`, the test will break.  `nock` does not introduce that issue.

Using `nock`, it would look something like this:

```js
test('searchConcept handles paging correctly.', async (t) => {
  const body1 = '{"feed":{"updated":"sometime","id":"someurl","title":"fake Cmr Results","entry":[{"cmrEntry1":"data1"}, {"cmrEntry2":"data2"}]}}';
  const body2 = '{"feed":{"updated":"anothertime","id":"another url","title":"more Results","entry":[{"cmrEntry3":"data3"}, {"cmrEntry4":"data4"}]}}';
  const body3 = '{"feed":{"updated":"more time","id":"yet another","title":"morer Results","entry":[{"cmrEntry5":"data5"}, {"cmrEntry6":"data6"}]}}';

  nock('https://cmr.uat.earthdata.nasa.gov')
    .get('/search/collections.json')
    .query((q) => q.page_num === '1')
    .reply(200, body1, { 'cmr-hits': 6 });

  nock('https://cmr.uat.earthdata.nasa.gov')
    .get('/search/collections.json')
    .query((q) => q.page_num === '2')
    .reply(200, body2, { 'cmr-hits': 6 });

  nock('https://cmr.uat.earthdata.nasa.gov')
    .get('/search/collections.json')
    .query((q) => q.page_num === '3')
    .reply(200, body3, { 'cmr-hits': 6 });

  const expected = [
    { cmrEntry1: 'data1' },
    { cmrEntry2: 'data2' },
    { cmrEntry3: 'data3' },
    { cmrEntry4: 'data4' },
    { cmrEntry5: 'data5' },
    { cmrEntry6: 'data6' }
  ];

  const results = await searchConcept('collections', {});

  t.deepEqual(expected, results);
});
```